app/assets/stylesheets/content/_wiki.sass

```diff
@@ -28,26 +28,48 @@
 
 div.wiki
   font-size: $wiki-default-font-size
-  $wiki-max-heading-font-size: 24px
-  line-height: 1.6em
+  $wiki-max-heading-font-size: $wiki-default-font-size * 1.5
 
-  // this should be the same as a normal heading (see toolbar)
+  // this needs to be the same as a normal level 2 heading (see toolbar)
   h1
     padding: 12px 0 0 0
     border: 0
     font-size: $wiki-max-heading-font-size
-    line-height: 1.4
+    line-height: $wiki-max-heading-font-size * 1.2
     margin-bottom: 16px
 
   h2
-    font-size: $wiki-max-heading-font-size * 0.8
+    $margin-top-size: $wiki-max-heading-font-size * 1.5
+    $margin-bottom-size: $margin-top-size * 0.6
+    margin: $margin-top-size 0 $margin-bottom-size 0 !important
+    font-size: $wiki-max-heading-font-size * 0.92
```

Comment:
There are a lot of magic numbers here. To reduce this I suggest to set the `line-height` to `normal`. Thus we have various line-heights depending on the font size, but less numbers. Further we do not need the extra space at the moment so `normal` should be sufficient. 
I really like your font-size adjustments. Before the headings were way to small. 👍 To make it however more convenient, a step size of five is probably better.